### PR TITLE
Add support for ccf flag for O11y

### PR DESCRIPTION
**Pull request by amaanbs**

### CODE REVIEW

File: bin/commands/runs.js [hunk 1]
```diff
@@ -61,6 +61,9 @@ module.exports = function run(args, rawArgs) {
     markBlockStart('setConfig');
     logger.debug('Started setting the configs');
 
+    // set cypress config filename
+    utils.setCypressConfigFilename(bsConfig, args);
```
Comment (karanshah-browserstack) [inline bin/commands/runs.js:65]: Let's move our code in end in this scenario?

**Review by karanshah-browserstack — COMMENTED** [review]
Can see this can impact other stuff too

**Review by 07souravkunda — APPROVED** [review]

File: bin/commands/runs.js (hunk 1 above)
Comment (amaanbs) [reply bin/commands/runs.js:65]: That would require moving more code around. This change shouldn't affect handling of other products.

**Review by rev-doshi — COMMENTED** [review]
This same changes is also needed for A11y AFAIR, can you test that once ?

**Comment by amaanbs** [comment]
> This same changes is also needed for A11y AFAIR, can you test that once ?
Tested, working fine.

**Review by RutvikChandla — APPROVED** [review]

**Review by pranavj1001 — APPROVED** [review]

**Pull request merged**